pybn/network.py

```diff
@@ -622,7 +622,7 @@ class Arc(object):
   def __init__(self, from_node, to_node):
     self.from_node = from_node
     self.to_node = to_node
-    self.to_node.addArcConnection(self.from_node.getName(),self.from_node.getIdNum(),self.from_node.getSize())
+    self.to_node.addArcConnection(self.from_node.getName(),self.from_node.getNodeId(),self.from_node.getSize())
```

Comment:
This changes what gets into the var of a node
This change might actually break how computeBeliefs work
Thus it is not necessary to make this change, even if redesigning computeBeliefs would be better in the long run